### DISCUSSION TITLE
Support for brotlicffi

### DISCUSSION THREAD

**Discussion by mgorny** [encode/httpx#1604] [General]
The old brotlipy project has been superseded by [brotlicffi](https://github.com/python-hyper/brotlicffi) to resolve the name collision with brotli bindings. Could you please add support for the new package?

**Comment by lovelydinosaur** [comment]
Thanks for the pointer. Resolved in #1605

**Comment by mgorny** [comment]
That was fast ;-). Thanks!

**Reply by lovelydinosaur** [reply]
The benefits of sponsored OSS. https://github.com/sponsors/encode
